ArjunaJTA/jta/classes/com/arjuna/ats/jta/logging/jtaI18NLogger.java

```diff
@@ -45,6 +45,9 @@
 /**
  * i18n log messages for the jta module.
  *
+ * For jtaI18NLogger, if any method, prefixed with <b>"warn_recovery"</b> is called then,
```

Comment:
We don't need this comment anymore. If you couple the functionality of the recovery problem reporting into the one place which is  the `XARecoveryModule` here then it's the responsibility of the person updating that class to consider it.
From my perspective there is no need to expose this responsibility to somebody adding a warn log message.

Reply:
okay